python-sdk/tests/benchmark/Makefile

```diff
@@ -27,8 +27,9 @@ clean:
 # Takes approximately 7min
 setup_gke: clean
 	@cd infrastructure/terraform && \
-		terraform init && \
-		terraform apply
+		terraform init -input=false && \
+		terraform plan -out=tfplan -input=false && \
+		terraform apply -input=false tfplan
```

Comment:
We can set an env var to disable input (see [here](https://www.terraform.io/cli/config/environment-variables#tf_input)). Then you don't need to pass it every time.

Reply:
Why do we need to use `-input-false`? Are there any variables that did not have their values set?

Reply:
Look like. otherwise, it throws an error 
```
Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: Releasing state lock. This may take a few moments...
Makefile:29: recipe for target 'setup_gke' failed
make: *** [setup_gke] Error 1
```